src/tar.ts

```diff
@@ -39,7 +53,14 @@ export async function extractTar(archivePath: string): Promise<void> {
     // Create directory to extract tar into
     const workingDirectory = getWorkingDirectory();
     await io.mkdirP(workingDirectory);
-    const args = ["-xz", "-f", archivePath, "-P", "-C", workingDirectory];
+    const args = [
+        "-xz",
+        "-f",
+        archivePath?.replace(/\\/g, "/"),
```

Comment:
`\` is a valid file character on UNIX systems, so we shouldn't replace it with a path separator if not on Windows.

We can either add a check for Windows here, or use `path.sep` to replace:
```
archivePath?.replace(path.sep, "/")
```
See https://nodejs.org/api/path.html#path_path_sep

Reply:
`replace(path.sep, "/")` only replaces the first instance of the separator. I did some googling and changed to `replace(new RegExp('\\' + path.sep, 'g'), "/")` instead.